### PR TITLE
Notification if admin is already busy answering Centcomm messages/Syndicate Messages/Nuke requests

### PR DESCRIPTION
**Pull request by Firecage**
Gives a message if an admin is already busy answering a Centcomm Message/Syndicate Message/Nuke Request.

At the request of sir Mc @Dorsisdwarf

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
Okand will have no more excuses. My mailed fist will be absolute

**Comment by Firecage** [comment]
I personally prefer Steel Plate Fists over Mail Fists.

**Comment by Dorsisdwarf** [comment]
Steel plate lowers my agility too much, makes it too hard to hit they greyshirts

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -1759,8 +1759,10 @@
 			usr << "The person you are trying to contact is not wearing a headset"
```
Comment (KorPhaeron) [inline code/modules/admin/topic.dm]: Missing period help

**Comment by Firecage** [comment]
Yes, but well made steel plate increases your armor class by a lot.

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (Firecage) [inline code/modules/admin/topic.dm]: @KorPhaeron So, you want to add periods where it is needed in these sentences? Okay.

**Pull request merged**